cmd/svcat/binding/describe_cmd_test.go

```diff
@@ -0,0 +1,106 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package binding
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/kubernetes-incubator/service-catalog/cmd/svcat/command"
+	"github.com/kubernetes-incubator/service-catalog/cmd/svcat/test"
+	"github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1beta1"
+	svcatfake "github.com/kubernetes-incubator/service-catalog/pkg/client/clientset_generated/clientset/fake"
+	"github.com/kubernetes-incubator/service-catalog/pkg/svcat"
+	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	k8sfake "k8s.io/client-go/kubernetes/fake"
+
+	_ "github.com/kubernetes-incubator/service-catalog/internal/test"
+)
+
+func TestDescribeCommand(t *testing.T) {
+	const namespace = "default"
+	testcases := []struct {
+		name         string
+		fakeBindings []string
+		bindingName  string
+		expected     string
+		wantError    bool
+	}{
+		{
+			name:         "describe non existing binding",
+			fakeBindings: []string{},
+			bindingName:  "mybinding",
+			expected:     "unable to get binding '" + namespace + ".mybinding': servicebindings.servicecatalog.k8s.io \"mybinding\" not found",
```

Comment:
Doing an exact full string match is going to make these tests difficult to maintain going forward. We have another set of tests that do full output comparisons on stdout/stderr for the cli.

I recommend that for these tests, we do a `strings.Contains` check instead, and only specify a small portion of the expected output. That will give us enough to check for error messages, without needing to regularly update these tests as the output changes.

So you could change this to just check for the portion of the error message that is controlled by the CLI: `unable to get binding ns/name`.

```
expected: "

Reply:
I agree and latest commit should make the tests work as suggested.